ui-library-filesharing-ui-components/app/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Can we take the time to clean up this README?

I imagine loading up a Microsoft sample and seeing a different readme as odd.